lend/src/components/ActionModal/Hero.tsx

```diff
@@ -523,6 +524,20 @@ const Hero: React.FC<HeroProps> = ({
     }
   }, [amountStr, pool, usdc]);
 
+  const getReferrerAddressToUse = async () => {
```

Comment:
We do because `getReferrerAddressToUse` does an async call for us from Airtable, without which `handleConfirm`  won't know which referral address to use. Do you think we should still make it a hook instead?